code/mobility-services/mobility-services.css

```diff
@@ -228,6 +237,53 @@ a.big-button-container {
 a.small-button {
   text-decoration: none;
 }
+/****************************************/
+/*** Dropdown Menu Buttons Navigation ***/
+/****************************************/
+
+/* hide custom menu if on mobile */
+@media (max-width: 799px) {
+  #tia-menu-list {
+    display: none;
+  }
+}
+
+/* hide knack menu buttons if on desktop */
+@media (min-width: 799px) {
```

Comment:
you might want to double check this pixel width, Knack doesnt always display mobile correctly at 799px. Im using 767.9px & 768px in ROW Portal right now. you should see the hamburger menu in the top right on mobile